package.json

```diff
@@ -33,6 +33,7 @@
     "@atlaskit/toggle": "12.0.3",
     "@atlaskit/tooltip": "17.1.2",
     "@jitsi/js-utils": "1.0.6",
+    "@material-ui/core": "4.11.3",
```

Comment:
No update to package-lock.json? We need to check that in.

Reply:
Yes indeed. 
We should also try to enforce a rule team wise to use the latest npm version (or at least the latest major version).  I've seen quite often the package-lock version changed from version 1 to 2 and vice versa. I guess this applies to the pipeline version too.